guides/release/in-depth-topics/autotracking-in-depth.md

```diff
@@ -321,4 +321,84 @@ class ShoppingList {
 }
 ```
 
+## Caching of tracked properties
+
+In contrast with computed properties from pre-Octane, autotracked properties
+are not cached. This means that an autotracked property will always be
+recomputed, regardless of whether its dependencies have changed. The following
+example shows how this works:
```

Comment:
I think it's better that we don't use the term `autotracked property`, in case this may cause a reader to think it's different from `tracked property`.
```suggestion
In contrast to computed properties from pre-Octane, tracked properties are
not cached. A tracked property can also be recomputed even though its
dependencies haven't changed. The following example shows this behavior:
```

Reply:
Makes sense.